server/service/src/sync/remote_data_synchroniser.rs

```diff
@@ -19,9 +19,18 @@ use repository::{
 
 use thiserror::Error;
 
+const INITIALISATION_POLL_PERIOD_SECONDS: u64 = 15;
+// 30 minutes
+const INITIALISATION_TIMEOUT_SECONDS: u64 = 30 * 60 * 60;
```

Comment:
30 minutes is 30 * 60 seconds though?

Reply:
oops, thanks @mark-prins: https://github.com/openmsupply/open-msupply/pull/1865/commits/f71fd72bbbf18ec977ac8bda4552fb66b520c775